apps/crn-frontend/src/shared-research/api.ts

```diff
@@ -170,3 +176,33 @@ export const getResearchTags = async (
 
   return response?.items || [];
 };
+
+export const getGeneratedOutputContent = async (
```

Comment:
is it possible to move some of these repeated functions to frontend-utils?

Reply:
Yea, that's a good suggestion, thanks 🤝😊. Working on that now